server/models/models.js

```diff
@@ -11,7 +11,36 @@ class UserModel {
     this.phonenumber = user.phonenumber;
   }
 
+  async signUp() {
+    const text = `INSERT INTO users (firstname, lastname,
```

Comment:
Hi DC, variable name text is not descriptive enough. Do consider using meaningful names